### DISCUSSION TITLE
PiMox HAOS VM - whiptail not installed on armbian default

### DISCUSSION THREAD

**Discussion by ustix** [tteck/Proxmox#2406] [Ideas]
I don't know if anybody else has the same problem, but whiptail is not a default package for a lot of linux systems, so it is not installed on armbian (not in minimail and not in standard). So I think it should be good if it the scripts check for whiptail, because without that, the script is not working.

**Comment by tteck** [comment]
Do you have the Proxmox port installed?

**Reply by ustix** [reply]
Yes, as the title states it is pimox. Arm64 version. It is working for me after installing whiptail package, but would be great if a dependency check would be included because it exits without saying anything if it is not.

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/commit/2a2dd14c9a4dd6622f8196193451e360d1978255
